app2/tsconfig.json

```diff
@@ -15,6 +15,7 @@
     "moduleResolution": "bundler",
     "allowImportingTsExtensions": true,
     "plugins": [
+      { "name": "@effect/language-service" },
```

Comment:
`tsconfig`s do not deep merge when extending. This must be duplicated because we can't properly extend `tsconfig.base.json` _and_ the config generated by Svelte.